src/sentry/conf/server.py

```diff
@@ -99,6 +99,8 @@ def env(
 
 ENVIRONMENT = os.environ.get("SENTRY_ENVIRONMENT", "production")
 
+IS_PRODUCTION = IS_PROD = ENVIRONMENT == "production"
+
```

Comment:
we don't need N ways to spell this -- please reuse `settings.DEBUG` -- it's what it is for